packages/ethereum/deploy/slimDeployments.ts

```diff
@@ -35,6 +35,8 @@ const main: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
     const contractName = contract.replace('.json', '')
     const compilerData =
       (await hre.artifacts.getBuildInfo(`contracts/${contractName}.sol:${contractName}`)) ?? data.compilerData
+    // sometimes not all contracts are deployed, depends on the deployment scripts
+    if (!compilerData) continue
```

Comment:
Why not all the contracts are deployed? Is this some behavior in the release?

Reply:
this is needed because we deploy the dummy proxy contract on all networks except xdai and the HoprNetworkRegistry only on xdai